.github/workflows/benchmark.yml

```diff
@@ -52,17 +52,16 @@ jobs:
                   docker compose -f docker-compose.dev.yml up -d
 
             - name: Set up Python
-              uses: actions/setup-python@v4
+              uses: actions/setup-python@v5
               with:
                   python-version: 3.10.10
+                  cache: 'pip'
+                  cache-dependency-path: 'requirements*.txt'
```

Comment:
Examples always include `**/` eg `**/requirements*.txt`
We probably don't need that?

Reply:
I wondered about that... we're super unlikely to add more requirements files but 🤷 

I guess I can check if it affects the running speed and (how much could it after all) it it isn't slower then it helps the future traveller to have the wildcard

Reply:
yeah it depends on how deep the nested folders go since it is a mono repo, but likely it won't affect anything noticeable, your call :D